### PR TITLE
Feature/update infrastructure

### PR DESCRIPTION
**Pull request by rainer-gepardec**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -25,16 +25,19 @@ IMPORTANT: When running in an IDE the actual port will depend on your configurat
...
 
 IMPORTANT: In this mode you won't be able to hot-code deploy.
 
 == Running in an IDE
 
 Refer to the documentation of the IDE of your choice, which must provide integration for Wildfly/JBoss servers, for how to set up the project.
 
-IMPORTANT: You need to run the server with the `standalone-microprofile.xml` configuration provided in the `infrastructure/wildfly` directory. +
-Copy this file into your wildfly `standalone/configuration` directory and add `-c standalone-microprofile.xml` to the startup script of your wildfly run configuration.
+Configure you wildfly to use microprofile configuration by adding `-c standalone-microprofile.xml` to your startup script in your run configuration.
```
Comment (cchet) [inline README.adoc:36]: your

Comment (cchet) [reply README.adoc:36]: Can't the Wildfly be configured via a maven command as well?

**Review by cchet — COMMENTED** [review]
Update the infrastructure doc (WIKI) as well

https://gepardec.github.io/microprofile-training/#infrastructure-guide

You find the markdown file here `docs/infrastructure-guide.md`

File: infrastructure/prometheus/prometheus.yml [hunk 2]
```diff
@@ -26,4 +26,4 @@ scrape_configs:
```
Comment (SabrinaPloner) [inline infrastructure/prometheus/prometheus.yml:26]: I think 'prometheus:19090' needs to be changed to 'prometheus:9090' (with 19090 I get 'connection refused' in Prometheus).

Comment (rainer-gepardec) [reply infrastructure/prometheus/prometheus.yml:26]: fixed

**Pull request merged**